internal/slackerror/errors.go

```diff
@@ -711,6 +713,18 @@ Otherwise start your app for local development with: %s`,
 		Message: "Failed to get teams for restricted user",
 	},
 
+	ErrFeedbackNameInvalid: {
+		Code:        ErrFeedbackNameInvalid,
+		Message:     "Invalid feedback name provided",
+		Remediation: fmt.Sprintf("View the feedback options with %s", style.Commandf("feedback --help", false)),
+	},
+
+	ErrFeedbackNameRequired: {
+		Code:        ErrFeedbackNameRequired,
+		Message:     "Please provide a feedback --name flag or remove the --no-prompt flag",
+		Remediation: fmt.Sprintf("View feedback options with %s", style.Commandf("feedback --help", false)),
+	},
```

Comment:
```suggestion
	ErrFeedbackNameRequired: {
		Code:    ErrFeedbackNameRequired,
		Message: "The name of feedback to provide is required",
		Remediation: strings.Join([]string{
			"Please provide a feedback --name flag or remove the --no-prompt flag",
			fmt.Sprintf("View feedback options with %s", style.Commandf("feedback --help", false)),
		}, "\n"),
	},
```

Small suggestion to restate the code as a `message` sentence and move the kind suggestion to remediation! 💌

Reply:
A quick preview in a text format 📸 

```
$ slack feedback --no-prompt

🚫 The name of feedback to provide is required (feedback_name_required)

💡 Suggestion
   Please provide a feedback --name flag or remove the --no-prompt flag
   View feedback options with `lack feedback --help`

```

Reply:
Great suggestion! I'll bring in your suggestion and follow-up with a commit that updates the `Message` of both `ErrFeedbackNameInvalid` and `ErrFeedbackNameRequired` to be consistent.

Reply:
Commit 0126938 updates the `Message` titles to be consistent